uwuno/src/main/java/com/learning/uwuno/cards/sColorCard.java

```diff
@@ -0,0 +1,22 @@
+package com.learning.uwuno.cards;
+
+// Class handles all special cards that are not black
+public class sColorCard implements card {
```

Comment:
Nvm. Too much work to go back now.

Reply:
I changed to all extend card, if were to do it your way I think basic card would have to extend sColorCard since basic cards have a value variable and I thought it was weird to make a special card the base class?